util/ubuntu-setup

```diff
@@ -92,11 +123,40 @@ if $YES; then
     ARGS="-y"
 fi
 
-PACKAGES="$(sed 's/#.*//;/^$/d' $BASE/util/ubuntu-packages.txt)"
+TOOLCHAIN_PACKAGES="libnewlib-dev libnewlib-arm-none-eabi gcc-arm-none-eabi libstdc++-arm-none-eabi-newlib"
+if [ "ubuntu-18.04" = "$SYSTEM" ]; then
+	# check for missing compiler
+	ARM_COMPILER=true
+	if [[ $(apt-cache policy gcc-arm-none-eabi) = *"Installed: (none)"* ]]; then
```

Comment:
Checking if the package is installed is probably better done by inspecting the exit code of

dpkg -s gcc-arm-none-eabi 

It seems to work for me at least :P